src/pymodule.cpp

```diff
@@ -12,12 +13,24 @@ using tesseract::PageSegMode;
 using tesseract::ResultIterator;
 using tesseract::TessBaseAPI;
 
+static void SetImageChecked(TessBaseAPI& api, Pix* image) {
```

Comment:
Minor: We don't have a strict code format standard in this repo yet. However, I think it is quite often to see function name start with lowercase letters.

Reply:
:+1: let me change that here and in the other PRs